### ISSUE TITLE
[KSwitch]: Add ariaLabelledBy prop

### ISSUE THREAD

**Issue by AlexVelezLl** [learningequality/kolibri-design-system#806]
## Product
Kolibri

Related: https://github.com/learningequality/kolibri/issues/12743

## Desired behavior

In some ocasions we want to have the switch input and its label with a different layout that wouldnt be possible to achieve with the current `label` prop. For example:

![Image](https://github.com/user-attachments/assets/c368e619-ace0-40d0-bfda-545b4c2fc8ae)

And for these cases we should have an `aria-labelledby` attribute in the `input` element, and this pointing to the element that has the desired label for the switch. So we should add a new prop to KSwitch named `ariaLabelledBy` that sets the `aria-labelledby` attribute of this input element.

## Current behavior
Currently we have a `label` prop/slot, but this is not enough for more complex layouts.

## The Value Add
Improved accessibility.

# Add labels
Please choose the appropriate label(s) from our existing label list to ensure that your issue is properly categorized. This will help us to better understand and address your issue!

**Comment by Sahil-Sinha-11** [comment]
Hey @AlexVelezLl  I would like to work on this issue.

**Comment by AlexVelezLl** [comment]
Hey @Sahil-Sinha-11! For sure! I will assign this issue to you :)

**Comment by Sahil-Sinha-11** [comment]
Hey @AlexVelezLl  I added the id and arialabelledBy here 
![Image](https://github.com/user-attachments/assets/cee6007e-abea-44a9-a890-6763ad897c0a)


passed it as aprop
![Image](https://github.com/user-attachments/assets/9dcec6bd-9261-4de2-8cbf-d3783424a259)


and genrated the id here
![Image](https://github.com/user-attachments/assets/eb1ba931-a577-4b45-bea3-8e3c55de6a01)

It gives some error, is this the right way to do it?
![Image](https://github.com/user-attachments/assets/7b2d2ecf-868f-4446-8240-0eb8dc3a201a)

**Comment by AlexVelezLl** [comment]
Hey @Sahil-Sinha-11! No, this aria-labelledBy is for cases where we will not be using the `label` that is rendered by the KSwitch but rather by an external component. So we dont need to implement a `labelId` computed prop, nor setting this value to the label or any div tag. We just need to set the `aria-labelledby` input attribute to the new prop `ariaLabelledBy`.

So we rather should do:
```Vue
  <input
    ...
    :aria-labelledby="ariaLabelledBy"
  />
```

**Comment by Sahil-Sinha-11** [comment]
Thanks for the guidance @AlexVelezLl. 
 Should it be done this way?
![Image](https://github.com/user-attachments/assets/3cc20b7e-da51-45ad-9bdb-2658cdb94532)
![Image](https://github.com/user-attachments/assets/fa69f02c-d833-47bb-b1ae-bb9ae7ce66ac)

**Comment by AlexVelezLl** [comment]
Yes! @Sahil-Sinha-11

**Comment by Sahil-Sinha-11** [comment]
Hey @AlexVelezLl I made a pr.

**Comment by AlexVelezLl** [comment]
Thank you @Sahil-Sinha-11! Will review it soon 🤗.

**Comment by AlexVelezLl** [comment]
Closed by #808
